pyproject.toml

```diff
@@ -5,9 +5,9 @@ description = "Nautobot Secrets Providers App"
 authors = ["Network to Code, LLC <opensource@networktocode.com>"]
 license = "Apache-2.0"
 readme = "README.md"
-homepage = "https://github.com/nautobot/nautobot-plugin-secrets-providers"
-repository = "https://github.com/nautobot/nautobot-plugin-secrets-providers"
-keywords = ["nautobot", "nautobot-plugin"]
+homepage = "https://github.com/nautobot/nautobot-app-secrets-providers"
+repository = "https://github.com/nautobot/nautobot-app-secrets-providers"
+keywords = ["nautobot", "nautobot-plugin", "nautobot-app"]
```

Comment:
Shall we remove "nautobot-plugin" from the keyword?

Reply:
Leaving for searchability.